java/src/shared/utils/FileUtils.java

```diff
@@ -0,0 +1,31 @@
+package shared.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class FileUtils {
+
+    public static final Map<String, String> MIMETYPES = makeMimeTypes();
+
+    private static Map<String, String> makeMimeTypes() {
```

Comment:
See [this SO answer](http://stackoverflow.com/a/847849/1530134) on how to normally do this. `makeMimeTypes` and `getMimeType` is not very clear, but whatevs